submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/notifications/NotificationsSubscriber.kt

```diff
@@ -22,29 +26,52 @@ internal class NotificationsSubscriber(
     private val subscriptionService: SubscriptionService,
     private val resourceLoader: ResourceLoader,
     private val userPreRegister: Observable<UserRegister>,
-    private val passwordReset: Observable<PasswordReset>
+    private val passwordReset: Observable<PasswordReset>,
+    private val successfulSubmission: Observable<SuccessfulSubmission>,
+    private val notificationProperties: NotificationProperties
 ) {
-
     @PostConstruct
     fun activationSubscription() {
         val template = ActivationTemplate(getTemplateContent("activation.html"))
         val subscription = NotificationType(EMAIL_FROM, "BioStudies account activation", template)
-        subscriptionService.create(subscription, userPreRegister.map { asNotification(it) })
+        subscriptionService.create(subscription, userPreRegister.map { asActivationNotification(it) })
     }
 
-    private fun asNotification(source: UserRegister) =
-        Notification(source.user.email, ActivationModel(FROM, source.activationLink, source.user.fullName))
-
     @PostConstruct
     fun passwordReset() {
         val template = PasswordResetTemplate(getTemplateContent("reset-password.html"))
         val subscription = NotificationType(EMAIL_FROM, "BioStudies password reset", template)
-        subscriptionService.create(subscription, passwordReset.map { asNotification(it) })
+        subscriptionService.create(subscription, passwordReset.map { asPasswordResetNotification(it) })
     }
 
-    private fun asNotification(source: PasswordReset) =
+    @PostConstruct
+    fun successfulSubmission() {
+        if (notificationProperties.submissionNotification) {
```

Comment:
I believe there is no scenario when we do not want email notification, we may want to configure allow to configure this at the user level but not  system level

Reply:
It doesn't make much sense to me to have this enabled in dev and beta since these are used mostly for testing purposes.

Reply:
Just wanted to add that from experience, sometimes we do need to turn of notifications globally before reloading part of the database automatically (usually in bulk).